autoscheduler/frontend/src/components/SchedulingPage/ConfigureCard/ConfigureCard.tsx

```diff
@@ -25,15 +25,20 @@ const ConfigureCard: React.FC = () => {
   const [includeFull, setIncludeFull] = React.useState(false);
   const [loading, setLoading] = React.useState(false);
   const [showSnackbar, setShowSnackbar] = React.useState(false);
-  // Holds a reference to the DOM element to check if the component is still mounted
-  const ref = React.useRef();
+
   const courseCards = useSelector<RootState, CourseCardArray>((state) => state.courseCards);
   const term = useSelector<RootState, string>((state) => state.term);
   const avsList = useSelector<RootState, Availability[]>((state) => state.availability);
   const dispatch = useDispatch();
 
+  // Holds a reference to the DOM element to check if the component is still mounted
```

Comment:
So what does mounted mean again?

Reply:
![](https://i2.wp.com/programmingwithmosh.com/wp-content/uploads/2018/10/Screen-Shot-2018-10-31-at-1.44.28-PM.png?ssl=1)

Basically, mounting happends when React intializes the component, and unmounting happens when the component's about to be destroyed and garbage-collected or whatever